microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -228,100 +409,47 @@ The updated `manifest.json` that includes the `content-scripts` and `web_accessi
-
 
 <!-- ------------------------------ -->
-#### Add jQuery
-
-This step has already been done in [jquery.min.js](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/lib/jquery.min.js) in the MicrosoftEdge-Extensions repo.
+#### The manifest's `content_scripts` section injects JavaScript into all browser tabs
 
-In the content scripts that you're injecting, plan on using jQuery (`$`).  You added a minified version of jQuery and put it in your extension package as `lib\jquery.min.js`.
+In the `content_scripts` key section, the `matches` attribute is set to `<all_urls>`, which means that `.js` (or `.css`) files that are specified in In the `content_scripts` key section are injected into all browser tab pages when each tab is loaded.  The allowed files types that can be injected are JavaScript (`.js`) and CSS (`.css`).  
 
-These content scripts run in individual sandboxes, which means that the jQuery injected into the `popup.js` page isn't shared with the content.
+In this sample, the following files are specified:
+* `content-scripts/content.js`
+* `lib/jquery.min.js`
 
 
 <!-- ------------------------------ -->
-#### Understanding the thread
-
-Even if the browser tab has JavaScript running on it on the loaded web page, any content that's injected doesn't have access to that JavaScript.  The injected JavaScript only has access to the actual DOM that's loaded in that browser tab.
-<!-- todo: mention 'thread', since heading says 'thread' -->
+#### How jQuery support is added (`jquery.min.js`)
 
+`jquery.min.js` is a predefined, minified file to support jQuery expressions starting with `$` in `popup.js` and `content.js`.
 
-<!-- ====================================================================== -->
-## Step 6: Add the content script message listener
-
-To continue manually building-up the Part 2 demo, you would need to create this file.  This step has already been done in [content.js](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/content-scripts/content.js) in the MicrosoftEdge-Extensions repo.
+In injected content scripts, it's common to use jQuery (`$`).  This sample includes a minified version of jQuery, residing in the extension package<!-- todo: check 'package' --> as `lib\jquery.min.js`.
 
-Here's the `content-scripts\content.js` file that gets injected into every browser tab page based on the `content-scripts` section in `manifest.json`:
+Each content script (`popup.js` and `content.js`) runs in an individual sandbox, which means that the jQuery that's injected into the initial popup page isn't shared with the current webpage.
```

Comment:
@captainbrosset pls critically review the wording in these two h4 sections:

https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/extension-samples/microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md#how-jquery-support-is-added-jqueryminjs

Each content script (popup.js and content.js) runs in an individual sandbox, which means that the jQuery that's injected into the initial popup page isn't shared with the current webpage.

Even if the browser tab has JavaScript running on it on the loaded web page, any content that's injected doesn't have access to that JavaScript. The injected JavaScript only has access to the actual DOM that's loaded in that browser tab.

Reply:
result:
```
The content script (`content.js`) runs in its own sandbox, which means that the jQuery that's injected into `content.js` isn't shared with the current webpage.
...
Even if the browser tab contains a webpage that runs JavaScript code, the `content.js` script that's injected in that browser tab by the extension doesn't have access to that JavaScript code.  The injected content script only has access to the DOM of the webpage.
```